media/js/base/mozilla-notification-banner-init.js

```diff
@@ -7,65 +7,39 @@ $(function() {
-        {
-            'id': 'fx-out-of-date-banner-copy1-direct-1',
-            'name': 'fx-out-of-date',
-            'experimentName': 'fx-out-of-date-banner-copy1',
-            'experimentVariant': 'direct-1',
-            'heading': 'Your browser security is at risk.',
-            'message': 'Update Firefox now to protect yourself from the latest malware.',
-            'confirm': 'Update now',
-            'confirmAction': 'Update Firefox',
-            'confirmLabel': 'Firefox for Desktop',
-            'url': '/firefox/new/?scene=2',
-            'close': 'Close',
-            'closeLabel': 'Close'
-        },
-        {
-            'id': 'fx-out-of-date-banner-copy1-direct-2',
-            'name': 'fx-out-of-date',
-            'experimentName': 'fx-out-of-date-banner-copy1',
-            'experimentVariant': 'direct-2',
-            'heading': 'Your Firefox is out-of-date.',
-            'message': 'Get the most recent version to keep browsing securely.',
-            'confirm': 'Update Firefox',
-            'confirmAction': 'Update Firefox',
-            'confirmLabel': 'Firefox for Desktop',
-            'url': '/firefox/new/?scene=2',
-            'close': 'Close',
-            'closeLabel': 'Close'
-        },
-        {
-            'id': 'fx-out-of-date-banner-copy1-foxy-1',
-            'name': 'fx-out-of-date',
-            'experimentName': 'fx-out-of-date-banner-copy1',
-            'experimentVariant': 'foxy-1',
-            'heading': 'Psst… it’s time for a tune up',
-            'message': 'Stay safe and fast with a quick update.',
-            'confirm': 'Update Firefox',
-            'confirmAction': 'Update Firefox',
-            'confirmLabel': 'Firefox for Desktop',
-            'url': '/firefox/new/?scene=2',
-            'close': 'Close',
-            'closeLabel': 'Close'
-        },
-        {
-            'id': 'fx-out-of-date-banner-copy1-foxy-2',
-            'name': 'fx-out-of-date',
-            'experimentName': 'fx-out-of-date-banner-copy1',
-            'experimentVariant': 'foxy-2',
-            'heading': 'Time to browse better!',
-            'message': 'Get the latest version of Firefox for extra speed and safety.',
-            'confirm': 'Update now',
-            'confirmAction': 'Update Firefox',
-            'confirmLabel': 'Firefox for Desktop',
-            'url': '/firefox/new/?scene=2',
-            'close': 'Close',
-            'closeLabel': 'Close'
-        }
-    ];
+    var headingText;
+    var messageText;
+    var confirmText;
+    var closeText;
+
+    // try to get localized copy
+    if (typeof utils !== 'undefined') {
+        headingText = utils.trans('global-fx-out-of-date-banner-heading');
+        messageText = utils.trans('global-fx-out-of-date-banner-message');
+        confirmText = utils.trans('global-fx-out-of-date-banner-confirm');
+        closeText = utils.trans('global-close');
+    // fall back to English copy if we can't get to the trans function for some reason
```

Comment:
Would it be better not to show the notification at all, rather than show untranslated English strings? The way the lib works, if any of the required things are not supplied, the notification should not show at all anyway.